### ISSUE TITLE
Missing duplex maxima in mensural

### ISSUE THREAD

**Issue by rettinghaus** [music-encoding/music-encoding#989]
Currently there is no value for a `duplex maxima` note (basically consisting of a pair of maxima notes) that can be found often in prints from the first half of the 17th century:

![image](https://user-images.githubusercontent.com/7693447/187928289-769e894d-65fb-459e-8f11-a0f7783c08cf.png)

![image](https://user-images.githubusercontent.com/7693447/187928426-19ae8e27-e474-4b13-96aa-f2a9c2e3c8a3.png)

![image](https://user-images.githubusercontent.com/7693447/187928622-dbd19c34-97a2-4795-bd1f-3b8c523bf0cc.png)

![image](https://user-images.githubusercontent.com/7693447/187928731-c9ec2175-02c6-4b86-99a3-bf7959efe0f6.png)

**Comment by bwbohl** [comment]
@music-encoding/ig-mensural as we already have a lot of changes concerning mensural notation since MEI 4.0.1 could you please confirm the need of this and potentially provide a pull request to add this? Adding this to [MEI 5.0.0 milestone](https://github.com/music-encoding/music-encoding/milestone/8)

**Comment by musicEnfanthen** [comment]
Pinging @music-encoding/ig-mensural @annplaksin @giulianodibacco

**Comment by annplaksin** [comment]
I would appreciate more information on that topic to decide on that to discuss an appropriate approach.
Is it really an additional note value and would require a new duration value?
Are these two notes and the latter one is flipped?
And what encoding purpose should the feature support?

**Comment by craigsapp** [comment]
I am wondering if these are only at the end of the music or major sections of the music, functioning as stylized terminal longs, which behaves like a fermata with an undetermined duration, or can they occur in the middle of the music (their duration being important as a double maxima to keep the part aligned with other parts).

Here is an example of the typesetter not caring about the exact (visual) duration of notes at the end:

https://polona2.pl/item/melodiae-na-psalterz-polski,Nzk3MTM4ODQ/34/#info:metadata

<img width="202" alt="Screenshot 2023-05-25 at 9 42 33 AM" src="https://github.com/music-encoding/music-encoding/assets/3487289/169ddcda-14b6-48e7-943c-1c0f76f847cc">

Note that there are nine minims (half notes) duration for the pitches before the last note , so clearly all of the last notes are supposed to start at the same time, and there is no reason for the last notes to have duration differences from each other.

One ending note looks like a long and the others like maxima, and one of the maxima having a reversed stem (which has no meaning).  Also one of the notes does not have a fermata (because the accidental is displayed instead).

It would be interesting to see the wider context of these notes as @annplaksin suggests.  It would be interesting to know if it is specific to a particular publisher or groups of localized publishers, and any manuscripts of the same music to see what they use (often the terminal long can be quite decorative, and sometimes elongated to fill in space.  In other words, these duplex maximas could be the printers attempt to such ornamentation of the terminal long in the manuscript they were transcribing, and they don't necessarily intend it to mean a duplex maximum but rather are trying to reflect the ornamentation of the handwritten version.   In this case it could be called a duplex maxima, but the gestural duration is not intended to be that of a duplex maxima.

**Comment by martha-thomae** [comment]
After the ODD Meeting, it seems that Craig is right, this `duplex maxima` pointed out by Klaus acts similar to the longas or maximas with a fermata (i.e., you hold it until the other voices end), but it is a particular symbol used in German printings. I am thinking that, given that basically these are two maximas combined (called "duplex maxima"), we could encode this with the pre-existing attributes and values we have. We could use `<note dur="maxima" dur.quality="duplex"/>`. What do you think?

**Comment by martha-thomae** [comment]
Only one thing I am realizing is that `duplex` is supposed to behave as `num="1" numbase="2"` (one in the place of two, so twice as long)... but we can always use the actual `num` and `numbase` pair of attributes to rewrite the note duration (as `num` and `numbase` are meant to overwrite the `dur.quality`).

So, we could encode the symbol (the "duplex maxima") as `<note dur="maxima" dur.quality="duplex"/>` and add the `@num` and `@numbase` attributes to provide the right duration for that symbol.

**Comment by martha-thomae** [comment]
I guess what I meant to say is that the encoding will be a bit inconsistent with how we encode other note symbols in that we usually use `dur` for encoding the symbols and `dur.quality` (or `num` and `numbase`) to encode their actual duration (e.g., we use `<note dur="longa" dur.quality="duplex">` to encode a duplex longa in Ars antiqua, a longa that doubles its regular duration). And in this "new" case we will be using both `dur` and `dur.quality` to encode the symbol (the duplex maxima) and then use `num` and `numbase` to overwrite its duration.

But at the same time we are not introducing more values for the `dur` attribute. I think it still is a workable solution.

You can also think that `dur` and `dur.quality` together are encoding a note class (e.g., a _duplex longa_, a _minor semibreve_, a _major semibreve_, an _altered breve_, an _imperfect breve_, etc.), which of course they say something about the duration of the note, but they are still different note classes, and that the "duplex maxima" is just another note class.

Anyway, you have read already too much from my side. I would like to hear/read what you think.

**Comment by bwbohl** [comment]
If I gather correctly from the above, it would be sufficient to add a duplex maxima encoding to the guidelines, correct?

**Comment by martha-thomae** [comment]
Yes, that would be correct. @rettinghaus could you add this example of the guidelines where you see fit, please?
@annplaksin : just tagging you to update you.

**Comment by rettinghaus** [comment]
Two points from my side:

* `att.duration.quality` plays into the logical domain and until now `@dur.qualtiy` hadn't yet any visual affect, but the duplex maxima note has a special appearance (so this would effectively mix up the meanings of `@dur` and `@dur.quality`)
* `att.duration.quality` has currently the constraint that `duplex` is only allowed for `longa` notes, it had to be changed to be also valid for `maxima` notes

Just to make sure we're on the right track.

**Comment by martha-thomae** [comment]
First point:

> * `att.duration.quality` plays into the logical domain and until now `@dur.qualtiy` hadn't yet any visual affect, but the duplex maxima note has a special appearance (so this would effectively mix up the meanings of `@dur` and `@dur.quality`)

I agree with your first point, and maybe it would be good to bring this up as a discussion with the Mensural IG. I will reassume the co-chairing of the IG with Anna on July 1st, and we wanted to organize a meeting to touch base with everyone. I think we can bring this up at that meeting.

Second point:

> * `att.duration.quality` has currently the constraint that `duplex` is only allowed for `longa` notes, it had to be changed to be also valid for `maxima` notes

Yes, that is the case, it would be a small change in the schema. Let's meet first with the IG sometime after July 1st (not sure when, let me discuss it with Anna), and then (if approved) you can implement this change into the schema.

:)

**Comment by DILewis** [comment]
I guess if we're worried about `@dur.quality` being about something visual (and I think I'm a little worried about that too), is there a reason not to be using `@altsim`? Especially if the note duration is more fermata-like... For discussion, perhaps.
